easybuild/easyconfigs/s/STELLA/STELLA-1.05.00-CrayGNU-17.08-double.eb

```diff
@@ -0,0 +1,37 @@
+# jg (cscs)
+easyblock = 'CMakeMake'
+
+name = 'STELLA'
+version = '1.05.00'
+versionsuffix = '-double'
+
+homepage = 'https://github.com/MeteoSwiss-APN/stella'
+description = """STELLA"""
+
+toolchain = {'name': 'CrayGNU', 'version': '17.08'}
+toolchainopts = {'verbose': False}
+# can't clone private c2sm repo (+ no official release)
```

Comment:
Does this not work? 
```
https://github.com/C2SM-RCM/cosmo-pompa
```

Reply:
jenscscs has no access to private repo

Reply:
As agreed in offline-discussion, we want to store all here: 
/apps/common/UES/easybuild/sources

The question that remains to be discussed with crClim people is which version we are going to use and how they are going to be updated.